src/main/com/smithsmodding/armory/client/textures/MaterializedTextureCreator.java

```diff
@@ -124,17 +95,12 @@ public static void registerBaseTexture(@Nonnull Collection<ResourceLocation> loc
 
     /**
      * Actual construction method is called from the ForgeEvent system.
-     * This method kicks the creation of the textures of and provided a map to put the textures in.
+     * This method kicks the creation of the textures of and provided a map to register the textures in.
      *
      * @param event The events fired before the TextureSheet is stitched. TextureStitchEvent.Pre instance.
      */
     @SubscribeEvent(priority = EventPriority.LOW)
     public void createCustomTextures(@Nonnull TextureStitchCollectedEvent event) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)